dev_docs/tutorials/ci.mdx

```diff
@@ -29,28 +29,76 @@ Build documentation from the root `docs` folder.
 
 ### Labels
 
-Labels can be added to a pull request to run conditional pipelines.
+Labels can be added to a pull request to run conditional pipelines.  Build artifacts will be available on the "Artifacts" tab of the "Build Kibana Distribution and Plugins" step.
+
+#### `ci:all-cypress-suites`
+
+Some Cypress test suites are only run when code changes are made in certain files, typically files with overlapping test coverage.  Adding this label will cause all Cypress tests to run.
+
+#### `ci:build-all-platforms`
+
+Build Windows, macOS, and Linux archives.
+
+#### `ci:build-canvas-shareable-runtime`
+
+Build the Canvas shareable runtime and include it in the distribution.
+
+#### `ci:build-cdn-assets`
+
+Build an archive that can be used to serve Kibana's static assets.
+
+#### `ci:build-cloud-image`
+
+Build cloud Docker images that can be used for testing deployments on Elastic Cloud.
+
+#### `ci:build-os-packages`
+
+Build Docker images, and Debian and RPM packages.
+
+#### `ci:build-serverless-image`
+
+Build serverless Docker images that can be used for testing deployments on Elastic Cloud.
+
+#### `ci:build-storybooks`
+
+Build and upload storybooks.
+
+#### `ci:build-webpack-bundle-analyzer`
+
+Build and upload a bundle report generated by `webpack-bundle-analyzer`.
 
 #### `ci:cloud-deploy`
 
-Create or update a deployment on Elastic Cloud.
+Create or update a deployment on Elastic Cloud production.
+
+#### `ci:cloud-persist-deployment`
+
+Prevents an existing deployment from being shutdown due to inactivity.
 
 #### `ci:cloud-redeploy`
 
 Create a new deployment on Elastic Cloud.  Previous deployments linked to a pull request will be shutdown and data will not be preserved.
 
-#### `ci:build-all-platforms`
+#### `ci:collect-apm`
 
-Build Windows, macOS, and Linux archives.  Artifacts will be available on the "Artifacts" tab of the "Build Kibana Distribution and Plugins" step.
+Collect APM metrics, available for viewing on the Kibana CI APM cluster.
 
-#### `ci:build-os-packages`
+#### `ci:no-auto-commit`
 
-Build Docker images, and Debian and RPM packages.  Artifacts will be available on the "Artifacts" tab of the "Build Kibana Distribution and Plugins" step.
+Skip auto-committing changed files.
 
-#### `ci:build-cloud-image`
+#### `ci:project-deploy-elasticsearch`
 
-Build Docker images that can be used for testing deployments on Elastic Cloud in the CFT region (gcp-us-west2).  Artifacts will be available on the "Artifacts" tab of the "Build Kibana Distribution and Plugins" step.
+Create or update an Elasticsearch Serverless project on Elastic Cloud QA.
 
-#### `ci:all-cypress-suites`
+#### `ci:project-deploy-observability`
+
+Create or update an Observability project on Elastic Cloud QA.
+
+#### `ci:project-deploy-security`
+
+Create or update a Security project on Elastic Cloud QA.
```

Comment:
Do you think these should mention serverless?

Reply:
:+1: I pushed 5206c088f18f5c8c1a24392caf97b917ad0d3b7d